File: QTD.SPA/src/app/_DtoModels/ExternalClassImport/ValidateCSV_ILA_Results_VM.ts
import { ValidateCSV_Results_VM } from "@models/ImportCSV/ValidateCSV_Results_VM";
import { ImportDatum_ILA_VM } from "./ImportDatum_ILA_VM";

export class ValidateCSV_ILA_Results_VM extends ValidateCSV_Results_VM{
    data: ImportDatum_ILA_VM[];
  }

File: QTD2.Infrastructure/Model/ILA/ValidateCSV_ILA_Results_VM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QTD2.Infrastructure.Model.Import_CSV_VM
{
    public class ValidateCSV_ILA_Results_VM : ValidateCSV_Results_VM
    {
        public List<ImportDatum_ILA_VM> Data { get; set; }
    }
}
